includes/discovery/sensors/dbm/timos.inc.php

```diff
@@ -47,3 +47,49 @@
         discover_sensor($valid['sensor'], 'dbm', $device, $oid, 'tx-' . $index, 'timos', $descr, $divisor, $multiplier, $limit_low, $warn_limit_low, $warn_limit, $limit, $value, 'snmp', $entPhysicalIndex, $entPhysicalIndex_measured, $user_func);
     }
 }
+//new code follows for multi-lane optics
+
+$numlanes = snmpwalk_cache_multi_oid($device, 'tmnxPortEntry', [], 'TIMETRA-PORT-MIB', 'timos');
+$lanes = snmpwalk_cache_multi_oid($device, 'tmnxDDMLaneEntry', [], 'TIMETRA-PORT-MIB', 'timos');
+
+foreach ($numlanes as $index => $entry) {
```

Comment:
Why do we walk tmnxPortTable first and not just walk tmnxDDMLaneTable?

If you think we need to walk tmnxPortTable, you really only use tmnxPortSFPNumLanes, so just walk that.